cuda/test/matrix/sellp_kernels.cpp

```diff
@@ -207,7 +183,8 @@ TEST_F(Sellp, AdvancedApplyWithSliceSizeAndStrideFActorIsEquivalentToRef)
 
 TEST_F(Sellp, SimpleApplyMultipleRHSIsEquivalentToRef)
 {
-    set_up_apply_matrix();
+    set_up_apply_matrix(64);
+
```

Comment:
Why did you add an additional empty line? The additional one should be removed (same with all following additions of empty lines after `set_up_apply_matrix`).